### DISCUSSION TITLE
###CATEGORY_ICON### is not working?

### DISCUSSION THREAD

**Discussion by MindstoneMedia** [oliverklee/ext-seminars#2902] [General]
Hello,
i have created categories with icons. In the details view I can see the icons. I inserted a new Marker ###CATEGORY_ICON### (copied from details view) in the subpart ###SINGLE_CATEGORY_ITEM###. But in the Frontend I see just the Marker?

What's wrong with my setup? Any Ideas?

Thank you
Oliver

<img width="543" alt="Bildschirmfoto 2024-02-04 um 17 24 43" src="https://github.com/oliverklee/ext-seminars/assets/74834981/52823fbb-b530-4ba4-b82a-5e41a8592618">

**Comment by oliverklee** [comment]
This might be a bug in the extension or an issue in your template.

Does is work if you use the HTML template provided by the extension?

**Reply by MindstoneMedia** [reply]
in the original template File is no marker ###CATEGORY_ICON### - so for testing I have insert the missing marker and I have the same effect.

**Reply by oliverklee** [reply]
Ah, I see. In general, if things aren't in the original HTML template, they most probably are not supported. So, in other words: Most probably category icons in the single view are not supported. (I don't remember exactly, though, and haven't tried it out myself.)
